app/src/main/java/com/trunghoang/aroundhere/data/db/PlaceLocalDataSource.java

```diff
@@ -63,4 +74,28 @@ public void getVisitedPlaces(@NonNull OnDataLoadedCallback<List<Place>> callback
         }, mPlaceDAO, callback);
         daoTask.execute();
     }
+
+    public void savePlace(SearchParams searchParams, @NonNull OnDataLoadedCallback<Place> callback) {
```

Comment:
method save place thì đầu vào nên là 1 place chứ em nhỉ??
```suggestion
    public void savePlace(Place place, @NonNull OnDataLoadedCallback<Place> callback) {
```